.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: docker.mirror.hashicorp.services/circleci/golang:1.15
+      - image: docker.mirror.hashicorp.services/circleci/golang:1.16.7
```

Comment:
FWIW I think 1.16.8 is out now. Do we need to pin to a specific patch version or would `golang:1.16` suffice?

Reply:
I was following Vault core, they pin to the patch. I am okay to pin to the minor, but then we will have variation if ever a build is rerun.